### ISSUE TITLE
Add support for <Code> tag in View XML files

### ISSUE THREAD

**Issue by brentonhouse** [brentonhouse/titanium-turbo#41]
> Add support for tag in View XML files that would simply output the code from the body of the element into the auto-generated controller.

JIRA Ticket:  https://jira.appcelerator.org/browse/ALOY-1700

#### Code can be added by `src` attribute:

```xml
<Code src="/code/myfile.js" />
```

#### By body text:

```xml
<Code>
// this is a test
var a = 123;
</Code>
```

#### Or by both:

```xml
<Code src="/code/myfile.js">
// this is a test
var a = 123;
</Code>
```

In the last case, it will put the code from the source file above the code from the body text.

The code will then be added to the auto-generated code that is put into the JavaScript controller.

**Comment by brentonhouse** [comment]
Created JIRA Ticket: https://jira.appcelerator.org/browse/ALOY-1700

**Comment by brentonhouse** [comment]
Created Alloy PR:  https://github.com/appcelerator/alloy/pull/937

**Comment by brentonhouse** [comment]
Merged PR #43
